### ISSUE TITLE
API Overview grid has an extra column

### ISSUE THREAD

**Issue by ig-ealbert** [Infragistics/convertwire#75]
Open xamtree-api-overview.adoc in Livewire.
Compare with the online help:
http://help.infragistics.com/doc/Silverlight?page=xamTree_API_Overview.html

There seem to be 3 columns in Livewire, but only 2 in the online help.

**Comment by ig-ealbert** [comment]
Also happens in compression-api-overview.adoc
http://help.infragistics.com/doc/Silverlight?page=Compression_API_Overview.html

**Comment by craigshoemaker** [comment]
These were remedied by a previous commit. :grinning:

**Comment by craigshoemaker** [comment]
still an issue per @ig-ealbert

**Comment by craigshoemaker** [comment]
@ig-ealbert perhaps you were looking at some old files? I just confirmed that the converted topics only have two columns here.

**Comment by ig-ealbert** [comment]
Looks like it's not actually a new column, just a lot of `&nbsp;`.  These occur in English too, but I don't know why they are not rendered there?  The XML is structurally the same between the languages.  Why are `&nbsp;` removed in English and not in Japanese .adoc files?

```
&lt;TR&gt;
&lt;TD&gt;&lt;STRONG&gt;xamTree Namespaces&lt;/STRONG&gt;&lt;/TD&gt;
&lt;TD&gt;&lt;STRONG&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp; Key Classes&lt;/STRONG&gt;&lt;/TD&gt;&lt;/TR&gt;
```

```
&lt;TR&gt;
&lt;TD&gt;&lt;STRONG&gt;xamTree 名前空間&lt;/STRONG&gt;&lt;/TD&gt;
&lt;TD&gt;&lt;STRONG&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp; キー クラス&lt;/STRONG&gt;&lt;/TD&gt;&lt;/TR&gt;
```

**Comment by craigshoemaker** [comment]
Most excellent catch @ig-ealbert! I added support to trim whitespace off bold and italicized content and added a rule to replace multiple occurrences of `$nbsp;` **only in table cells** with just one space. Closed by ec6a957f7ec9820531201bf21050adee4c91a9bb
